### ISSUE TITLE
Having Issue with package

### ISSUE THREAD

**Issue by wrteamshakir** [iexbase/tron-api#174]
I installed it Successfully , but when i try  to install another  packages of laravel , it throws me error 
Problem 1
    - comely-io/data-types[1.0.10, ..., 1.0.33] require php ^7.1 -> your php version (8.1.12) does not satisfy that requirement.
    - comely-io/data-types 1.0.34 requires php ^7.2 -> your php version (8.1.12) does not satisfy that requirement.
    - iexbase/tron-api v5.0 requires comely-io/data-types ^1.0 -> satisfiable by comely-io/data-types[1.0.10, ..., 1.0.34].
    - iexbase/tron-api is locked to version v5.0 and an update of this package was not requested.

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.

as removing this tron api package , the errors gets disappear , please help me to solve it

**Comment by farshidpg** [comment]
same issue  , is there any solution ?

I check the ```comely-io/data-types```  and its deprecated

**Comment by wrteamshakir** [comment]
i can't see any solution regarding this , yes it is deprecated and there is not replacement of it .

**Comment by farshidpg** [comment]
I read package code and just once used one of methods of ```comely-io/data-types``` I think if ```iexbase/tron-api``` owner  want can change and update code but I think ```iexbase/tron-api``` is dead ... because there is no support , no update  for 8 months ... even install with ```--ignore-platform-reqs``` and ``` --ignore-platform-reqs -W``` no help to install in real project with other composer packages , just use in fresh empty project without any package that don't have conflicts

**Comment by wrteamshakir** [comment]
yes i have installed it and after installing another package it will not allow me because of `comely-io/data-types` .
